clientconn.go

```diff
@@ -75,8 +75,6 @@ var (
 	// errConnClosing indicates that the connection is closing.
 	errConnClosing = errors.New("grpc: the connection is closing")
 	errNoAddr      = errors.New("grpc: there is no address available to dial")
-	// minimum time to give a connection to complete
-	minConnectTimeout = 20 * time.Second
```

Comment:
I haven't look the following. But why did u delete this? This is defined at https://github.com/grpc/grpc/blob/master/doc/connection-backoff.md.